components/Caretaker.js

```diff
@@ -5,58 +5,71 @@ import Times from './Times';
 import AddTask from './AddTask';
 import normalize from 'react-native-normalize';
 import { ScrollView } from "react-native-gesture-handler";
+import {fetcher} from '../API/API.js';
 
-const Caretaker = ({history, navigation}) => {
-  const [timers, editTimers] = useState([
-    {id: 1, name: "Wake-up", time: new Date(), window: new Date()},
-    {id: 2, name: "Take meds", time: new Date(), window: new Date()},
-    {id: 3, name: "go to sleep", time: new Date(), window: new Date()}
-  ])
-  const deleteTimer = (id) => {
-    editTimers(prevTimers => {
-      return prevTimers.filter(timer => timer.id != id)
-    })
+
+class Caretaker extends React.Component {
```

Comment:
I think `Caretaker`  being a class component was a good choice. The state can now hold the tasks and reflect changes made from the receiver end.